src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -595,20 +575,20 @@ The following list contains the different calls you can make with the API, inclu
 
     <CollapserGroup>
       <Collapser
-        id=""
-        title="AcceptDistributedTraceHeaders"
+        id="InsertDistributedTraceHeaders"
+        title="InsertDistributedTraceHeaders"
       >
         ### Syntax
 
         ```cs
-        void AcceptDistributedTraceHeaders(carrier, getter, transportType)
+        void InsertDistributedTraceHeaders(carrier, setter)
         ```
 
-        Instrument the called service for inclusion in a distributed trace.
+        Adds [distributed tracing](/docs/apm/agents/net-agent/configuration/distributed-tracing-net-agent/#manual-instrumentation) data to an outbound message to another instrumented service.
 
         ### Description
 
-        `ITransaction.AcceptDistributedTraceHeaders` is used to links the spans in a trace by accepting a payload generated by [`InsertDistributedTraceHeaders`](/docs/apm/agents/net-agent/net-agent-api/net-agent-api/#InsertDistributedTraceHeaders) or generated by some other W3C Trace Context compliant tracer. This method accepts the headers of an incoming request, looks for W3C Trace Context headers, and if not found, falls back to New Relic distributed trace headers. This method replaces the deprecated [`AcceptDistributedTracePayload`](/docs/apm/agents/net-agent/net-agent-api/net-agent-api/#AcceptDistributedTracePayload) method, which only handles New Relic distributed trace payloads.
+        `ITransaction.InsertDistributedTraceHeaders` modifies the carrier object that is passed in by adding W3C Trace Context headers and New Relic Distributed Trace headers. The New Relic headers can be disabled with `<distributedTracing excludeNewrelicHeader="true" />` in the config.
```

Comment:
add contraction "that is" should be "that's"

Distributed Trace headers should be lowercased, since they aren't in code format